k8s/upgrades/0.8.0-0.8.1/cstor_pool_upgrade.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/bash
+set -e
+
+##################################################################
+# STEP: Get SPC name as argument                                 #
+#                                                                #
+# NOTES: Obtain the pool deployments to perform upgrade operation#
+##################################################################
+
+function usage() {
+		echo
+		echo "Usage:"
+		echo
+		echo "$0 <spc-name> <openebs-namespace>"
+		echo
+		echo "  <spc-name> Get the SPC name using: kubectl get spc"
+		echo "  <openebs-namespace> Get the namespace where pool pods"
+		echo "    corresponding to SPC are deployed"
+		exit 1
+}
+
+if [ "$#" -ne 2 ]; then
+    usage
+fi
+
+spc=$1
+ns=$2
+
+##Checking the version of OpenEBS ####
+openebs_version=$(kubectl get csp -o jsonpath="{.items[0].metadata.labels.openebs\.io/version}")
+if [ $openebs_version != "0.8.0" ]; then
+		echo "Current replica version is not 0.8.0";exit 1;
```

Comment:
What will be the message if similar thing is done with CVR ? 
So I would put the message like : 
`"Current cstor pool version is not 0.8.0"`